.github/ISSUE_TEMPLATE/issue_template_des.md

```diff
@@ -0,0 +1,10 @@
+# Title
+
+One to two sentence description.
+
+## Type of Issue (Design)
+
+- [ ] User research/testing
+- [ ] Wireframing
+- [ ] Greyscales
+- [ ] Hi-Fi Mockup
```

Comment:
@nataliesvoboda @AncaBalaceanu what do you guys think about this as the template for when designers create issues in ZenHub?